schemas/IDLs/org.gel.models.metrics.avro/1.0.1/ReportedVsGenetic.avdl

```diff
@@ -0,0 +1,153 @@
+General information about the checks, versions, tools, and number of markers
+TODO: Consider removing Inferred karyotype
+*/
+    record CoverageBasedSex {
+        string sampleId;
+        union {null, KaryotypicSex} inferredKaryotype;
+        union {null, double} ratioChrX;
+        union {null, double} ratioChrY;
+        union {null, double} avgCnvChrX; // should be here?
+        union {null, double} avgCnvChrY; // should be here?
+    }
+
+/**
+Mendelian inconsistencies
+========================================
+- fmendel reports a line per nuclear family
+- imendel reports a line per member of the family
+*/
+    record PerFamilyMendelErrors {
+        string fatherId;
+        string motherId;
+        union {null, double} numberOfOffspring;
+        union {null, double} numberOfMendelErrors;
+    }
+
+    record IndividualMendelErrors {
+        string sampleId;
+        union {null, double} numberOfMendelErrors;
+        union {null, double} rateMendelErrors;
+    }
+
+    record AggregatedIndividualMendelErrors {
+        string sampleId;
+        union {null, double} totalnumberOfMendelErrors;
+    }
+
+    record TotalNumberOfMendelErrors {
+        union {null, double} family;
+        union {null, array<AggregatedIndividualMendelErrors>} individual;
+    }
+
+    record LocusMendelSummary {
+        string sampleId;
+        string chr;
+        double code;
+        double numberOfErrors;
+    }
+
+    record MendelianInconsistencies {
+        union {null, array<PerFamilyMendelErrors>} perFamilyMendelErrors;
+        union {null, array<IndividualMendelErrors>} individualMendelErrors;
+        union {null, TotalNumberOfMendelErrors} totalNumberOfMendelErrors;
+        union {null, array<LocusMendelSummary>} locusMendelSummary;
+    }
+
+/** Relatedness
+========================================
+*/
+    record RelatednessPair {
+        string sampleId1;
+        string sampleId2;
+        union {null, double} ibd0;
+        union {null, double} ibd1;
+        union {null, double} ibd2;
+        union {null, double} piHat;
+    }
+
+    record FamilyRelatedness {
+        union {null, array<RelatednessPair>} relatedness;
+    }
+
+/** Evaluation
+========================================
+reportedVsGeneticSummary: familyPassesGvsRChecks, familyFailsACheck, familyMissingACheck
+*/
+    record CoverageBasedSexCheck {
+        string sampleId;
+        union {null, Sex} reportedPhenotypicSex;
+        union {null, KaryotypicSex} reportedKaryotypicSex;
+        union {null, KaryotypicSex} inferredSexKaryotype;
+        union {null, Query} sexQuery;
+    }
+
+    record MendelianInconsistenciesCheck {
+        string sampleId;
+        union {null, Query} mendelianInconsistenciesQuery;
+        union {null, string} comments;
+    }
+
+    record familyRelatednessCheck {
+        string sampleId1;
+        string sampleId2;
+        union {null, string} relationshipFromPedigree;
+        union {null, string} possibleRelationship;
```

Comment:
We better use an enum to list possible values. This is hard coded in tiering in something like
```
'Father'
'Mother'
'Son'
'Daughter'
'ChildOfUnknownSex'
'MaternalAunt'
'MaternalUncle'
'MaternalUncleOrAunt'
'PaternalAunt'
'PaternalUncle'
'PaternalUncleOrAunt'
'PaternalGrandmother'
'PaternalGrandfather'
'MaternalGrandmother'
'MaternalGrandfather'
'TwinsMonozygous'
'TwinsDizygous':
'TwinsUnknown'
'FullSiblingF'
'FullSiblingM'
```

But, this is legacy classification that don't make everybody happy, adapt as required.

Reply:
Yes, we use this in `relationshipFromPedigree`. I thought about making it an enum but I was afraid that we will add more relationships in the future. Happy to make it enum if we can easily extend the list afterwards.

Reply:
I think this would be good. Maybe adding a handy `Other` and/or `Unknown` alleviates the problem of the growing terms.
@Antonior26 ?

Reply:
Actually, @katherinesmith and I discussed with @Antonior26 about the possibility of extending the list to other relationships that we have observed but are not yet implemented in the tiering function. How easy would be to add new items to the enum in the future? We would prefer (if possible) to avoid `Other` or `Unknown`.